packages/utils/src/utils.ts

```diff
@@ -16,3 +16,5 @@ export const getVSColor = () => {
     a: 0.8,
   } as RGBA
 }
+
+export const SENTRY_DNS = 'https://6e86226331e84bd9885554fdac788ce7@o481102.ingest.sentry.io/5543775'
```

Comment:
I wonder if we should put this into a repo secret?

Reply:
I found this. But maybe we should to be safe.

https://docs.sentry.io/product/sentry-basics/dsn-explainer/#:~:text=DSNs%20are%20safe%20to%20keep,read%20access%20to%20any%20information.


<img width="754" alt="Screen Shot 2022-06-28 at 7 09 50 AM" src="https://user-images.githubusercontent.com/63378463/176200026-f2f6a8a9-1547-4a22-97f1-36b252112f2c.png">

Reply:
Thanks for checking, we can keep it public then.